.github/lineage.yml

```diff
@@ -0,0 +1,6 @@
+---
+version: "1"
+
+lineage:
+  skeleton:
+    remote-url: https://github.com/cisagov/skeleton-generic.git
```

Comment:
This line should be updated to use [cisagov/skeleton-python-library](https://github.com/cisagov/skeleton-python-library) as the parent repo.

Reply:
@jsf9k I made that fix here - https://github.com/cisagov/domain-manager-cli/pull/75/commits/d039a0c35e9871b5ba0f16d9dc4d928923e415ec